Resources/Prototypes/Entities/Objects/Devices/Electronics/door_access.yml

```
@@ -262,14 +297,16 @@
-      delta: AllAccess
-    # Sunrise-end
+  # Fire-start
+  alertAccesses:
+    scpBlue: ScpBlueAlertAccesses
+    scpGray: ScpGrayAlertAccesses
+    scpRed: ScpRedAlertAccesses
+    scpPurple: ScpPurpleAlertAccesses
+    scpBlack: ScpBlackAlertAccesses
+    scpDelta: ScpDeltaAlertAccesses
+    scpColdSilver: ScpColdSilverAlertAccesses
+  # Fire-end
```

YAMLlint (1.37.1)

[warning] comment not indented like content

(comments-indentation)